### PR TITLE
#12 implemented logout action

### LINKED ISSUE #12: Log out/Sign out Functionality
As an authenticated user, I want to be able to log out of mentorpaired application.

Add functionality for the user to logout. This process should take the user back to the login page.

### PR DESCRIPTION
**Pull request by fola-ooo**
Fixes #12

### CODE REVIEW

**Review by Nkoli — APPROVED** [review]
LGTM

**Review by toystars — CHANGES_REQUESTED** [review]
@FolaDee why is `logOutAction` a component?

Also, why is the `isLoggedOut` state in the `Logout` component and not part of the global redux state?

**Comment by fola-ooo** [comment]
> @FolaDee why is `logOutAction` a component?
> 
> Also, why is the `isLoggedOut` state in the `Logout` component and not part of the global redux state?

@toystars I made logOutAction a component because it was not clear to me what the content of the Dashboard component content is supposed to be and that was the only component with the Log out button which is in Page and Log out is not really a page and more of an action
I have made changes by making isLoggedOut part of the global redux

**Comment by toystars** [comment]
> > @FolaDee why is `logOutAction` a component?
> > Also, why is the `isLoggedOut` state in the `Logout` component and not part of the global redux state?
> 
> @toystars I made logOutAction a component because it was not clear to me what the content of the Dashboard component content is supposed to be and that was the only component with the Log out button which is in Page and Log out is not really a page and more of an action
> I have made changes by making isLoggedOut part of the global redux

@FolaDee my point is, you are placing a component in a file that is clearly named as an action. In redux, actions are usually `string` values dispatched to make changes to your state or run some side effects. That `logOutAction.js` file should be renamed to `Login.js` and moved to the components directory.

**Comment by fola-ooo** [comment]
> > > @FolaDee why is `logOutAction` a component?
> > > Also, why is the `isLoggedOut` state in the `Logout` component and not part of the global redux state?
> > 
> > 
> > @toystars I made logOutAction a component because it was not clear to me what the content of the Dashboard component content is supposed to be and that was the only component with the Log out button which is in Page and Log out is not really a page and more of an action
> > I have made changes by making isLoggedOut part of the global redux
> 
> @FolaDee my point is, you are placing a component in a file that is clearly named as an action. In redux, actions are usually `string` values dispatched to make changes to your state or run some side effects. That `logOutAction.js` file should be renamed to `Login.js` and moved to the components directory.

@toystars I have created a components directory and renamed the file as requested

File: src/components/Logout.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+class Logout extends Component {
+    render() {
+        const isLoggingIn = window.location.href.indexOf('/login') > -1;
+        const { isLoggedIn } = this.props;
+
+        if (!isLoggedIn) {
+            if (isLoggingIn) {
+                return <div> </div>
+            }
+            return <Redirect to="/signup" push={true} />
```
Comment (toystars) [inline src/components/Logout.js:15]: @FolaDee what is this supposed to do?

Comment (fola-ooo) [reply src/components/Logout.js:15]: It redirects to sign up if the user is not logged in

**Pull request merged**